src/main/java/software/sigma/bu003/internship/coparts/user/service/exception/UserAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package software.sigma.bu003.internship.coparts.user.service.exception;
+
+public class UserAlreadyExistsException extends RuntimeException{
+    public UserAlreadyExistsException(String message) {
+        super(message);
```

Comment:
you can make this message pretty - to add user name and email for example

Reply:
corrected